modules/weaviate/weaviate.go

```diff
@@ -5,10 +5,16 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/docker/go-connections/nat"
 	"github.com/testcontainers/testcontainers-go"
 	"github.com/testcontainers/testcontainers-go/wait"
 )
 
+const (
+	httpPort = nat.Port("8080/tcp")
```

Comment:
I have just only one final concern before merging this PR: do we need the dependency with nat here, or we can simply go with strings for the ports?

Not a blocker, as we already get that dependency from the core library, but wanted to double check with you first.

Reply:
I can remove this dependency if you want, no problem with that. I will change that to strings then